config.def.h

```diff
@@ -178,4 +178,12 @@ static const cursor_t imgcursor[3] = {
 	CURSOR_LEFT, CURSOR_ARROW, CURSOR_RIGHT
 };
 
+/* width of navigation area on left and right side of the window:
+ * a positive value defines the width as percentage of the window width,
+ * a negative value defines the width in pixels (without the negative sign),
+ * 0 disables cursor navigation,
+ * the area width will be kept under half of the window width in any case.
+ */
+static const int NAV_WIDTH = 33;
+
```

Comment:
I think the name `NAV_WIDTH` is okay, we can use `NAV_USE_PERCENT` for the bool switch.

```suggestion
static const bool NAV_USE_PERCENT = true; /* true means 33 will equal 33%, false means 33 will equal 33 pixels */
static const unsigned int NAV_WIDTH = 33;

```

Reply:
@OliverLew is this okay by you or are there any objection to this method?

Reply:
I think it's OK, it indeed makes the comments much shorter.

I used another name for switcher, the comment will tell user it's a percentage. Going to push the commit soon.